src/device/usbd.c

```diff
@@ -382,6 +388,12 @@ bool tud_connect(void) {
   return true;
 }
 
+bool tud_sof_cb_enable(bool en)
+{
+  usbd_sof_enable(_usbd_rhport, SOF_CONSUMER_USER, en);
+  return true;
```

Comment:
A return value makes no sense here.

While also true for my original PR that wasn't as obvious. I wasn't sure if every driver has implemented `dcd_sof_enable()` and so used `TU_VERIFY()` to check for that.

Reply:
You are right, forgot to change.